nimby-vs-yimby/README.md

```diff
@@ -0,0 +1,303 @@
+# NIMBY vs. YIMBY: outlier analysis of new construction projects in USA
+
+During the second iteration of DoltHub's USA housing price data bounty a large amount of public real estate data was scraped, wrangled and imported into a version-controlled database. This enables us to do some exploration and analysis for the purpose of gaining insight into the dynamics of real estate market. Some parts of United States are said to suffer from NIMBYism - a resistance to new property developments in area. One famous example is Marc Andreesen, a prominent Silicon Valley venture capitalist, going out of his way to prevent new housing to be built in his town - Atherton, CA. But perhaps there's also areas that welcome and support new real estate projects? By wielding the power of programming and open data, we are able to leverage the `us-housing-prices-v2` database and find which are which. 
+
+Our approach to data analysis approach is going to be as follows. We are going to limit data being analysed to timeframe from 2009 June 30 to 2020 January 1. This will provide us 10.5 years worth of data from the times between the official end of Great Recession of 2008 to the very beginning of the current quite complicated decade. Thus we will be looking into steady-state trends from relatively recent past period that had no major shocks to the entire real estate market. Furthermore, we are going to narrow our view into the initial records of property being sold, which implies that a property is newly built and just entered the market. We are going to count such initial sales for each county represented in the database. Some counties are geographically large, some are populous, some are small in area and/or population. To accomplish an apples to apples comparison we are going to normalise number of initial sales by population and by land area. Lastly, we are going to compute standard deviation values for each county from per capita and per area values to appreciate how much they stand out.
+
+Property sales data was primarily sources from the official county institution websites. Due to scale and complexity of the data landscape the gathered dataset is unfortunately quite sparse - only 417 counties and county-equivalent territories out of over 3000 are represented in the database for the time window we're looking into. This makes a general analysis for the entire USA impossible, but we're looking for places that stand out in terms of how little or how much new property objects are entering the market. Thus we will further narrow down the results to shortlist such outliers.
+
+We are going to use [US counties dataset](https://www.openintro.org/data/index.php?data=county_complete) provided by OpenIntro to enrich data scraped during bounty with population and land area numbers so that we could computer per capita and per area values. Once that is done, we are ready to crunch the numbers.
```

Comment:
```suggestion
We are going to use [US counties dataset](https://www.openintro.org/data/index.php?data=county_complete) provided by OpenIntro to enrich data scraped during the bounty with population and land area numbers so that we could computer per capita and per area values. Once that is done, we are ready to crunch the numbers.
```